updatinate.yml

```diff
@@ -20,14 +20,13 @@ php-net:
   download-url: http://php.net/distributions/php-{version}.tar.gz
 
 projects:
+  cos-php:
+    repo: git@github.com:pantheon-systems/cos-runtime-php.git
+    path: ${working-copy-path}/cos-php
+    #fork: git@github.com:pantheon-circleci/cos-runtime-php.git
```

Comment:
should this commented line be removed before merge?

Reply:
We decided to give the pantheon-upstream user write access to the cos-runtime-php repo, so we won't be using this for now.

I do wonder, though, if it wouldn't be better for ALL of the PRs to use "fork", and we currently have none using it. I think I'll keep this around for (light) documentation.